src/python/pants/backend/python/tasks2/python_binary_create.py

```diff
@@ -128,6 +131,12 @@ def _create_binary(self, binary_tgt, results_dir):
         dump_sources(builder, tgt, self.context.log)
       dump_requirements(builder, interpreter, req_tgts, self.context.log, binary_tgt.platforms)
 
+      # Dump built python distributions, if any, into builder's chroot.
+      built_dists = self.context.products.get_data(BuildLocalPythonDistributions.PYTHON_DISTS)
```

Comment:
If there are two local dists belong to two separate targets (say A and B).  Running `pants binary A B` should generate .pex only contain the right local dist.  Does this code handle the multiple targets scenario correctly?

Reply:
✅